app/AndroidManifest.xml

```diff
@@ -49,7 +51,7 @@
 
     <application
         android:allowBackup="false"
-        tools:replace="android:label,android:icon"
+        tools:replace="android:label,android:icon, android:allowBackup"
```

Comment:
The external library doesn't share our backup restriction but sets it, this overrides that flag to prevent a build conflict.